packages/taiko-client/driver/chain_syncer/blob/syncer.go

```diff
@@ -48,6 +49,7 @@ type Syncer struct {
 	reorgDetectedFlag   bool
 	maxRetrieveExponent uint64
 	blobDatasource      *rpc.BlobDataSource
+	mutex               *sync.Mutex
```

Comment:
```suggestion
	mutex               sync.Mutex
```

Reply:
Then don't need to add code `new(sync.Mutex)` at line93.

Reply:
Changed